setup.py

```diff
@@ -215,6 +215,9 @@ def read(*rnames):
 #   }
 }
 
+if 'RADICAL_DEBUG' in os.environ:
+    setup_args['install_requires'].append('setproctitle')
+
```

Comment:
Yes, that is true actually. We have this line for that:
https://github.com/radical-cybertools/radical.pilot/blob/devel/src/radical/pilot/bootstrapper/default_bootstrapper.sh#L61
but that needs updating I guess.  That does not have a conditional for `RADICAL_DEBUG` though.  I think I would like to let the `proctitle` stuff run for a while before we do that in non-debug mode.